examples/test_passphrase.cpp

```diff
@@ -0,0 +1,53 @@
+#define BOOST_TEST_MODULE passphrase_test
+#include <boost/test/included/unit_test.hpp>
+#include <boost/algorithm/string.hpp>
+#include <iostream>
+#include <thread>
+#include <chrono>
+#include <cli_widgets/hidden_input_guard.hpp>
+#include <boost/process.hpp>
+#include <boost/filesystem.hpp>
+
+ BOOST_AUTO_TEST_CASE(passphrase_test) {
+  namespace bp = boost::process;
+  namespace fs = boost::filesystem;
+
+  bp::opstream in;
+  bp::ipstream out;
+  bp::pipe p;
+  std::vector<std::string> outline;
+  std::string value;
+  std::string path_to_exe;
+
+  fs::path current_path = fs::absolute(fs::current_path());
+  for (fs::directory_entry &entry : fs::recursive_directory_iterator{current_path}) {
+    if (entry.path().filename().string() == "stdin_hide_guard" ||entry.path().filename().string() == "stdin_hide_guard.exe" ) {
+      path_to_exe = entry.path().string();
+    }
+  }
+
+  std::vector<std::string> many_password = {"Un p@ssw0rd balnéaire à Zürich ?"};
```

Comment:
Only one ?

Reply:
You can logically put as many as you want in the vector

Reply:
Which is what I’m asking for so that the test is more complete. Fuzzing would even be better.